Cubical/HITs/Hopf.agda

```diff
@@ -18,16 +38,286 @@ HopfS² (surf i j) = Glue S¹ (λ { (i = i0) → _ , idEquiv S¹
                                ; (j = i1) → _ , _ , rotIsEquiv (loop i) } )
 
 -- Hopf fibration using more direct definition of the rot equivalence
+-- TODO : prove that it is equivalent to HopfSuspS¹
 HopfS²' : S² → Set
 HopfS²' base = S¹
 HopfS²' (surf i j) = Glue S¹ (λ { (i = i0) → _ , rotLoopEquiv i0
                                 ; (i = i1) → _ , rotLoopEquiv i0
                                 ; (j = i0) → _ , rotLoopEquiv i0
                                 ; (j = i1) → _ , rotLoopEquiv i } )
 
--- Hopf fibration using suspension of S¹
-HopfSuspS¹ : SuspS¹ → Set
-HopfSuspS¹ north = S¹
-HopfSuspS¹ south = S¹
-HopfSuspS¹ (merid x i) = ua (_ , rotIsEquiv x) i
+-- Total space of the fibration
```

Comment:
Maybe this should be called `TotalHopf` instead to indicate what it is the total space of